src/security/BlindPeerConnector.h

```diff
@@ -9,6 +9,8 @@
 #ifndef SQUID_SRC_SECURITY_BLINDPEERCONNECTOR_H
 #define SQUID_SRC_SECURITY_BLINDPEERCONNECTOR_H
 
+#include "http/forward.h"
+#include "HttpRequest.h"
```

Comment:
It would be better to stop inlining the constructor instead, but I do not insist on this improvement.

Reply:
I'll skip for now, this PR is huge as it is. Noting it for a followup PR